PENDING.md

```diff
@@ -61,6 +61,12 @@ CLI flag.
 
 ### SDK
 
+* \#3750 Track outstanding rewards per-validator instead of globally,
```

Comment:
we should add the outstanding rewards endpoint removal on breaking changes too

Reply:
It has not been removed, just moved to per-validator (you have even more query granularity)

Reply:
Moved to Breaking Changes & added a bit on the REST endpoint change.